intel-vtune.spec

```diff
@@ -0,0 +1,25 @@
+### RPM external intel-vtune 2017.0.2.478468
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/intel-vtune.xml
+<tool name="intel-vtune" version="%{realversion}">
+  <info url="https://software.intel.com/en-us/intel-vtune-amplifier-xe"/>
+  <lib name="ittnotify"/>
+  <client>
+    <environment name="INTEL_VTUNE_BASE" default="/afs/cern.ch/sw/IntelSoftware/linux/x86_64/xe2017/vtune_amplifier_xe_%{realversion}"/>
+    <environment name="INCLUDE" default="$INTEL_VTUNE_BASE/include"/>
+    <environment name="LIBDIR" default="$INTEL_VTUNE_BASE/lib64"/>
```

Comment:
this directory contains libsdtc++. this can cause problems. Looks like vtune executables linked with full path e.g.
```
        libstdc++.so.6 => /afs/cern.ch/sw/IntelSoftware/linux/x86_64/xe2017/vtune_amplifier_xe_2017.0.2.478468/bin64/./../lib64/libstdc++.so.6 (0x00007f4003603000)
```
So I guess we can drop it and instead add
```
<flags LDFLAGS="$(INTEL_VTUNE_BASE)/lib64"/>
```